Revit_Core_Engine/Convert/Facade/FromRevit/FrameEdge.cs

```diff
@@ -43,8 +44,33 @@ public static partial class Convert
 
         public static oM.Facade.Elements.FrameEdge FrameEdgeFromRevit(this FamilyInstance familyInstance, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null)
         {
-            BH.Engine.Reflection.Compute.RecordError("Conversion of FrameEdges from Revit for the Facade discipline is not yet implemented.");
-            return null;
+            Mullion mullion = familyInstance as Mullion;            
+            if (mullion == null)
+                return null;
+
+            settings = settings.DefaultIfNull();
+
+            FrameEdge bHoMFrameEdge = refObjects.GetValue<FrameEdge>(mullion.Id);
+            if (bHoMFrameEdge != null)
+                return bHoMFrameEdge;
+
+            BH.oM.Geometry.ICurve location = mullion.LocationCurve.IFromRevit();
```

Comment:
I would put an extra null check, just in case
```suggestion
            BH.oM.Geometry.ICurve location = mullion.LocationCurve?.IFromRevit();
```

Reply:
Fixed in latest commit.